### PR TITLE
Minecraft Server app: Removes old caching, vertical center image, logic for validating fields

### PR DESCRIPTION
**Pull request by p0ns**
# Description
Fixes minecraft app

- Removes old caching
- vertical center image
- logic for validating fields
- default image 

# Copilot

### Summary
🌐🐛🎨

Improved the performance, reliability, and appearance of the `minecraft_server.star` app. Added HTTP cache to reduce network requests, added error handling for JSON parsing, and centered the text and icons on the display.

> _`minecraft_server.star` is the key to our fate_
> _We refactored the code to make it great_
> _We use the HTTP cache to speed up the load_
> _We handle JSON errors and center the mode_

### Walkthrough
*  Simplify and improve main function using HTTP cache feature, error handling, and render.Box ([link](https://github.com/tidbyt/community/pull/1909/files?diff=unified&w=0#diff-62501a98a72fdc222421ef479a88f7849a75088eb2b745a28d4e39674d7e4e8fL17-R59))
* Remove unused imports of cache.star and json.star modules ([link](https://github.com/tidbyt/community/pull/1909/files?diff=unified&w=0#diff-62501a98a72fdc222421ef479a88f7849a75088eb2b745a28d4e39674d7e4e8fL8-R8))

### CODE REVIEW

**Comment by p0ns** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/minecraftserver/minecraft_server.star [hunk 1]
```diff
@@ -5,75 +5,58 @@ Description: View Minecraft Server Activity and icon.
...
-            base64.encode(result.body()),
-            ttl_seconds = 300,
-        )
-        onlinePlayers = result.json()["players"]["online"]
-        maxPlayers = result.json()["players"]["max"]
-        motd = result.json()["motd"]["clean"][0]
-        motd2 = result.json()["motd"]["clean"][1]
-        iconURL = result.json()["icon"].split(",")[1]
+    apiURL = "".join(["https://api.mcsrvstat.us/2/", minecraftURL])
+    result = http.get(apiURL)
```
Comment (matslina) [inline apps/minecraftserver/minecraft_server.star:17]: If you still want the 300s TTL, pass `ttl_seconds=300` to `http.get()`.

**Review by matslina — APPROVED** [review]

**Comment by matslina** [comment]
@Cast324 FYI

**Comment by p0ns** [comment]
Added TTL and rebased :)

**Comment by p0ns** [comment]
I was wondering if there's anything else needed to merge this. Thanks!

**Pull request merged**